JOSESwift/Sources/CryptoImplementation/RSA.swift

```diff
@@ -45,34 +45,70 @@ fileprivate extension SignatureAlgorithm {
     }
 }
 
-fileprivate extension AsymmetricKeyAlgorithm {
+public extension AsymmetricKeyAlgorithm {
```

Comment:
@daniel-mohemian - I did try changing this back to `fileprivate` however, the build fails for the tests where i use that to iterate over the cases.  Even though I'm importing the `JOSESwift` modules as `@testable`.  I believe this is normal behavior for modules where one has to set things to `piublic` in order to access them outside of the module.  I've subsequently had to change it back to public.  I'm open to any other ideas you may have to resolve this.

Reply:
The `@testable` import gives you access to `public` (obviously) and `internal` access levels. If you change the extension to be `internal` it will work. 👌

Reply:
Changing it to `internal` fixed the issue.  ty for the assist.